### ISSUE TITLE
Android Submit program shouldn't complain about a lack of "test" directory

### ISSUE THREAD

**Issue by DavidWhitlock** [DavidWhitlock/PortlandStateJava#295]
For Winter 2020, I added logic to the `Submit` program to warn the submitter when they forget to submit their test code.  The submit program for Android projects doesn't correctly identify the test code in Android projects.  This was confusing to people.

```
*** WARNING: You are not submitting a "test" directory.
    Your unit tests are executed as part of the grading of your project. 
```

I should update the logic in the Android submit program to properly locate the test code.

**Comment by DavidWhitlock** [comment]
I should also make sure that the appropriate gradle files are submitted.  Some students did and some didn't.  It confused grading.

**Comment by DavidWhitlock** [comment]
Addressed in release 2021.4.1.
